src/commands/joinLeaveCommands.ts

```diff
@@ -5,4 +5,10 @@ export const abeLeaves = (member: GuildMember | PartialGuildMember) => {
         const { guild } = member;
         guild?.systemChannel?.send('https://media.giphy.com/media/fDO2Nk0ImzvvW/giphy.gif')
     }
+}
+
+export const newAccountJoins = (member: GuildMember | PartialGuildMember) => {
+    if(((new Date()).getTime() - member.user!.createdTimestamp) < 1000 * 60 * 5) {
+        member.send(`Hey ${member.user!.username} - just a reminder, your account needs to be at least 5 minutes old to chat`)
```

Comment:
probably could update this to put an actual channel tag in there but let's just add this for now

```suggestion
        member.send(`Hey ${member.user!.username} - just a reminder, your account needs to be at least 5 minutes old to chat. While you wait, feel free to browse our welcome channel for some basic rules and channel descriptions.`)
```